eventhubs/data-plane/pom.xml

```diff
@@ -1,79 +1,66 @@
-	
-	<modelVersion>4.0.0</modelVersion>
-	
-	<groupId>com.microsoft.azure</groupId>
-	<artifactId>azure-eventhubs-clients</artifactId>
-	<version>2.0.0</version>
-	<packaging>pom</packaging>
-	
-	<url>https://github.com/Azure/azure-event-hubs</url>
-	
-	<properties>
-		<proton-j-version>0.31.0</proton-j-version>
-		<qpid-proton-j-extensions-version>1.1.0</qpid-proton-j-extensions-version>
-	  	<junit-version>4.12</junit-version>
-		<slf4j-version>1.8.0-alpha2</slf4j-version>
-	</properties>
-	
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.5</version>
-				<configuration>
-					<source>1.8</source>
-					<target>1.8</target>
-					<optimize>true</optimize>
-					<showDeprecation>true</showDeprecation>
-					<showWarnings>true</showWarnings>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
-	<repositories>
-		<repository>
-			<id>HTTP proxy server in java</id>
-			<url>http://raw.github.com/SreeramGarlapati/jproxy/master/releases</url>
-		</repository>
-	</repositories>
-	 <dependencies>
-         <dependency>
-             <groupId>org.apache.qpid</groupId>
-             <artifactId>proton-j</artifactId>
-             <version>${proton-j-version}</version>
-         </dependency>
-         <dependency>
-             <groupId>com.microsoft.azure</groupId>
-             <artifactId>qpid-proton-j-extensions</artifactId>
-             <version>${qpid-proton-j-extensions-version}</version>
-         </dependency>
-         <dependency>
-             <groupId>org.slf4j</groupId>
-             <artifactId>slf4j-api</artifactId>
-             <version>${slf4j-version}</version>
-         </dependency>
-         <dependency>
-             <groupId>junit</groupId>
-             <artifactId>junit</artifactId>
-             <version>${junit-version}</version>
-             <scope>test</scope>
-         </dependency>
-		 <dependency>
-			 <groupId>org.jutils.jproxy</groupId>
-			 <artifactId>jproxy</artifactId>
-			 <version>0.0.1</version>
-			 <scope>test</scope>
-		 </dependency>
-	 </dependencies>
-	
-	 <modules> 
-	    <module>azure-eventhubs</module>
-	    <module>azure-eventhubs-eph</module>
-	    <module>azure-eventhubs-extensions</module>
-	 </modules>
+  <name>Microsoft Azure Event Hubs SDK Parent</name>
+  <description>Java libraries for talking to Windows Azure Event Hubs</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../../pom.client.xml</relativePath>
+  </parent>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+  <properties>
```

Comment:
these values are already specified in the parent pom.client.xml?

Reply:
Should I remove it? for all eventhubs' child modules? It is kind of conflict what I know from previous PR from Jonathan.
@JonathanGiles

Reply:
Connie's point is that these version numbers may be (or are) already specified in a parent pom, and therefore inherited here. You should try to not duplicate version properties in children as this can lead to inconsistent versions being used. You should check to ensure that they only exist in one place - the root parent pom file - and remove them from all other places.

Reply:
ah, do you mean <version>1.0.0-SNAPSHOT</version> ?

Reply:
No - the content of the properties section that Connie has left her comment against.

Reply:
remove line 11:   <version>2.0.0</version>, right?

Reply:
i see, to delete the entire <properties>...</properties>